script/core/diagnostics/luacheck.lua

```diff
@@ -39,6 +29,59 @@ files.isLua = function(uri)
     return isLua(uri)
 end
 
+local defold_config = {
```

Comment:
Not really sure if we need this tbh? We get the globals from the editor. Not really any point in also including them here.

Reply:
I did note that what the editor provides also contains messages such as "set_camera" and it is missing the new "fixed_update" lifecycle function